pkgs/development/python-modules/segyio/default.nix

```diff
@@ -12,15 +12,7 @@
 
 stdenv.mkDerivation rec {
   pname = "segyio";
-  version = "1.9.9";
-
-  patches = [
-    # PR https://github.com/equinor/segyio/pull/531
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done in #230343. Sorry for this!